### DISCUSSION TITLE
change country selected from another input

### DISCUSSION THREAD

**Discussion by 1stevengrant** [goveo/react-international-phone#109] [General]
Is this doable?

I have a checkout page that has shipping address details first and when I change country, I'd like to change the phone country too.

**Comment by josephpalma** [comment]
@1stevengrant I believe the defaultCountry prop will suffice for your use case. `defaultCountry={country}` where `country` is your shipping address country.

**Comment by ybrusentsov** [comment]
@1stevengrant This is possible in v4, you can set country with the `ref.setCountry` function
https://react-international-phone.vercel.app/docs/Usage/PhoneInput#ref-forwarding

**Reply by ybrusentsov** [reply]
@1stevengrant Please ping me if it's not working as expected
